File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.py
N, X = map(int, input().split())
if (1 <= N) and (1 <= X) and (N <= 10000) and (X <= 10000):
    arr = list(map(int,input().split()))
    for i in arr:
        if(i < X):
            print(i, end="")
            print(" ", end="")

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.c
#include <stdio.h>
int main(void){
    int N, X, a, c = 0;
    int arr[10000];
    scanf("%d %d", &N, &X);
    if (1 <= N && 1 <= X && N <= 10000 && X <= 10000){
        for (int i = 0; i < N; i++){
            scanf("%d", &a);
            if(a<X){
                arr[c] = a;
                c++;
            }
        }
        for(int i=0; i<c; i++){
            printf("%d ", arr[i]);
        }
    }
}